### PR TITLE
26 constructor

### PR DESCRIPTION
**Pull request by BovkunLeonid**

### CODE REVIEW

File: Constructor/script.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    Accumulator.call(this, number);
+    this.numberStart = number,
+    this.clear = function() {
+        this.number = this.numberStart;
+    } 
+}
+
+CancelableAccumulator.prototype = Accumulator.prototype;
+
+let accumDuracell = new CancelableAccumulator (150);
```
Comment (RomanIlkiv) [inline Constructor/script.js:33]: should be no space after name of function:
`new CancelableAccumulator(150);`

File: Constructor/script.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+console.log(accumVarta.number);
+
+accumVarta.decrement();
+console.log(accumVarta.number);
+
+// Task 2 
+
+function CancelableAccumulator(number) {
+    Accumulator.call(this, number);
+    this.numberStart = number,
```
Comment (RomanIlkiv) [inline Constructor/script.js:25]: `;` instead of `,`

File: Constructor/script.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+// Task 1
+
+function Accumulator (number) {
+    this.number = number,
```
Comment (RomanIlkiv) [inline Constructor/script.js:4]: `;` instead of `,`

**Review by RomanIlkiv — APPROVED** [review]

File: Constructor/script.js (hunk 3 above)
Comment (BovkunLeonid) [reply Constructor/script.js:4]: Ок. 
Thank you

File: Constructor/script.js (hunk 2 above)
Comment (BovkunLeonid) [reply Constructor/script.js:25]: Ок. 
Thank you

File: Constructor/script.js (hunk 1 above)
Comment (BovkunLeonid) [reply Constructor/script.js:33]: Thank you

**Pull request merged**